study.md

```diff
@@ -49,10 +49,26 @@ A kitchen has many items.
 -   An herbs and spices rack containing chives, lemon grass, and mint on one
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
-```json
-<!-- your answer here -->
-```
-
+json
+**** I did my best below ****
+{
+	"kitchen": {
+		"cupboard": {
+			"bowls": 12,
+			"plates": 12
+		},
+		"spicerack1": {
+			"chives": true,
+			"lemon-grass": true,
+			"mint": true
+		},
+		"spicerack2": {
```

Comment:
Yup, this works. However, you could simply put them in an array if that's easier